From 45442432228b846faf3864623b206814fee34010 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Tue, 12 Oct 2021 12:15:52 +0800
Subject: [PATCH 1/4] Revert "dt-bindings: crypto: hse: add dt bindings
 documentation"

This reverts commit a4ab318c1b3e82d4c4487199c228328163e0dd83.

This patch is to make the pkcs11-hse userspace demo work normally.

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../devicetree/bindings/crypto/nxp-hse.yaml   | 74 -------------------
 1 file changed, 74 deletions(-)
 delete mode 100644 Documentation/devicetree/bindings/crypto/nxp-hse.yaml

diff --git a/Documentation/devicetree/bindings/crypto/nxp-hse.yaml b/Documentation/devicetree/bindings/crypto/nxp-hse.yaml
deleted file mode 100644
index 5c7ff062dbc6..000000000000
--- a/Documentation/devicetree/bindings/crypto/nxp-hse.yaml
+++ /dev/null
@@ -1,74 +0,0 @@
-# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
-%YAML 1.2
----
-$id: http://devicetree.org/schemas/crypto/nxp-hse.yaml#
-$schema: http://devicetree.org/meta-schemas/core.yaml#
-
-title: NXP Hardware Security Engine
-
-maintainers:
-  - Bogdan Folea <bogdan.folea@nxp.com>
-
-description: |
-  - NXP Hardware Security Engine is featured on some NXP S32 platforms:
-    Crpto offloading support is currently available on S32G274A and S32R45.
-
-properties:
-  compatible:
-    oneOf:
-      - enum:
-          - fsl,s32gen1-hse
-    description: Standard property. Must include "fsl,<soc_family>-hse".
-
-  reg:
-    maxItems: 2
-    description: Standard property. Should specify the physical address and size
-                 of MU hardware register space and service descriptor space.
-
-  reg-names:
-    maxItems: 2
-    description: Standard property. Should specify the names associated with
-                 the MU hardware register space and service descriptor space.
-
-  interrupts:
-    maxItems: 3
-    description: Standard property. Should specify the TX, RX and general
-                 purpose interrupt lines used by the respective MU instance.
-
-  interrupt-names:
-    maxItems: 3
-    description: Standard property. Should specify the names associated with
-                 the interrupt lines used by the respective MU instance.
-
-additionalProperties: false
-
-required:
-  - compatible
-  - reg
-  - reg-names
-  - interrupts
-  - interrupt-names
-
-examples:
-  - |
-    hse: crypto {
-        compatible = "simple-bus";
-        #address-cells = <2>;
-        #size-cells = <2>;
-        #interrupt-cells = <3>;
-        ranges;
-
-        mu0b@40210000 {
-                compatible = "fsl,s32gen1-hse";
-                reg = <0x0 0x40210000 0x0 0x1000>,
-                      <0x0 0x22C00000 0x0 0x1000>;
-                reg-names = "hse-mu0b-regs",
-                            "hse-mu0b-desc";
-                interrupts = <GIC_SPI 103 IRQ_TYPE_EDGE_RISING>,
-                             <GIC_SPI 104 IRQ_TYPE_EDGE_RISING>,
-                             <GIC_SPI 105 IRQ_TYPE_EDGE_RISING>;
-                interrupt-names = "hse-mu0b-ack",
-                                  "hse-mu0b-rx",
-                                  "hse-mu0b-err";
-        };
-    };
-- 
2.25.1

